common-tools/clas-utils/src/main/java/org/jlab/utils/system/ClasUtilsFile.java

```diff
@@ -37,32 +73,36 @@ public static void printLog(String log){
     public static String getResourceDir(String env, String rpath){
         
         String envString = System.getenv(env);
+
         if(envString==null){
             ClasUtilsFile.printLog("Environment variable ["+env+"] is not defined");
             envString = System.getProperty(env);
         }
         
         if(envString == null){
             ClasUtilsFile.printLog("System property ["+env+"] is not defined");
-            return null;
+            if (envString.equals("COATJAVA") || envString.equals("CLAS12DIR")) {
```

Comment:
`envString` is `null` here, did you mean `env.equals`?
```suggestion
            if (env.equals("COATJAVA") || env.equals("CLAS12DIR")) {
```